tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/attachment_options.py

```diff
@@ -9,10 +9,9 @@
 
 class AttachmentOptions(Base):
 
-    _attachment_options_locator = (By.CSS_SELECTOR, '#attachment-options[data-type="action"]')
-    _view_button_locator = (By.ID, 'attachment-options-view')
-    _header_locator = (By.CSS_SELECTOR, '.current gaia-header')
-    _cancel_button_locator = (By.ID, 'attachment-options-cancel')
+    _attachment_options_locator = (By.CSS_SELECTOR, 'form.visible[role="dialog"][data-subtype="menu"]')
+    _view_button_locator = (By.CSS_SELECTOR, '[role="dialog"][data-type="action"] > menu > button[data-l10n-id="view-attachment-image"]')
```

Comment:
After taking a look at the DOM, `button[data-l10n-id="view-attachment-image"]` seems to be unique and simpler